src/Sylius/Behat/Context/Api/Admin/ManagingCountriesContext.php

```diff
@@ -348,6 +348,17 @@ public function iShouldBeNotifiedThatNameOfTheProvinceIsRequired(): void
         );
     }
 
+    /**
+     * @Then I should be notified that provinces that are in use cannot be deleted
+     */
+    public function iShouldBeNotifiedThatProvincesThatAreInUseCannotBeDeleted(): void
+    {
+        Assert::contains(
+            $this->responseChecker->getError($this->client->getLastResponse()),
+            'Cannot delete, the province is in use.'
```

Comment:
This error message and `Error Cannot delete, the province is in use.` src/Sylius/Behat/Context/Ui/Admin/ManagingCountriesContext.php:284 shouldn't be more likely the same?

Reply:
"Error" before the error message is something that exists only in flash messages as a message header.
![image](https://user-images.githubusercontent.com/13316080/171833793-c760700c-95ed-4eed-bd7b-5db101ed6b03.png)

I think it's a good idea to compare just a message in the UI context but I would be for doing it in separate PR as it requires changes in many behat contexts.